packages/react-components/theme-designer/src/ThemeDesigner.tsx

```diff
@@ -17,17 +17,17 @@ export const ThemeDesigner: React.FC<ThemeDesignerProps> = props => {
 
   const [state, dispatchState] = useThemeDesignerReducer();
 
-  const overridenTheme = overrideTheme(state.theme, state.brand, state.overrides);
+  const themeOverride = themeOverrides(state.theme, state.brand, state.overrides);
```

Comment:
seems this function no longer needs theme. also, there's no need for the overrides to have a hex value. They could just be palette['40'] or whatever, and pull directly from the palette